apps/react/src/models/base-filter-params.ts

```diff
@@ -0,0 +1,23 @@
+/** Base filter params. */
+export namespace BaseFilterParams {
```

Comment:
You don't have filters here. I suggest renaming the type
```suggestion
export namespace BaseFetchParams {
```

Reply:
FetchParams is a great name for this case, but FilterParams is also provided by the mentors of the VN team. This is so controversial 😬